data_science_1/README.md

```diff
@@ -0,0 +1,93 @@
+### Overview
+
+Nos encontramos en un mundo ficticio en el cual han descubierto que una molécula hasta ahora desconocida, el **thanatol**, es un excelente predictor de mortalidad cardiovascular: altas concentraciones de thanatol en sangre están asociadas a altas tasas de infartos de miocardio.
+
+Tenemos los datos de un [estudio observacional](https://en.wikipedia.org/wiki/Observational_study) que pretende investigar la relación entre el ejercicio físico y el nivel de thanatol en sangre. La idea es investigar si hay recomendaciones que puedan hacer los médicos a sus pacientes en base a los resultados del estudio.
+
+Una base de datos de sqlite en el fichero [dataset_3.sql](dataset_3.sql) contiene los datos del estudio.
+
+Los datos de los participantes adultos se han recogido en las siguientes tablas:
+
+* `data_collection_day` contiene la fecha de la realización del estudio para los participantes adultos
+* `participants` contiene los ids de los participantes adultos
+* `ages` contiene las fechas de nacimiento de los participantes adultos
+* `exercise` contiene las horas de ejercicio por semana medias que realizan los participantes adultos
+* `thanatol` contiene el nivel de thanatol medido en sangre del paciente, en milimoles por litro
+
+Mientras que los datos de los participantes menores de edad se han recogido con otra metodología en la tabla `pediatric`, que contiene columnas:
+
+* `age`: edad en años en el momento del estudio (puede ser fraccional)
+* `hours_per_day`: horas de ejercicio por día medias que realiza el participante
+* `mmol_per_l`: el nivel de thanatol medido en sangre del paciente, en milimoles por litro
+
+#### ejercicio 1
+
+Queremos combinar los datos de los pacientes adultos y menores de edad para poder tratarlos de forma homogénea.
+
+Crea una view que contenga las siguientes columnas para todos los participantes, tanto menores como adultos:
+
+* age: edad del participante en años en el momento del estudio
```

Comment:
Minor formatting consistency issue about column names, eg: `age` will be consistent.